package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"devDependencies": {
-		"markdownlint": "^0.36.1",
+		"markdownlint": "^0.37.0",
 		"markdownlint-cli": "^0.43.0"
```

Comment:
@coderabbitai, After testing with the `markdownlint` `0.37.0` version installed, There does not seem to be any regression :shrug:


@dependabot merge